CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [next]
 
+- patch(): Avoid unwanted mutation in Group [#9151](https://github.com/fabricjs/fabric.js/pull/9151)
```

Comment:
I think the correct commit would be:
patch(Group): Avoid unwanted mutation to passed objects array to Group constructor